examples/learn/async/custom_client/decorator/mistral/base_message_param.py

```diff
@@ -1,7 +1,11 @@
+import os
+
 from mirascope.core import BaseMessageParam, mistral
-from mistralai.async_client import MistralAsyncClient
+from mistralai import Mistral
 
 
-@mistral.call("mistral-large-latest", client=MistralAsyncClient())
+@mistral.call(
+    "mistral-large-latest", client=Mistral(api_key=os.environ["MISTRAL_API_KEY"])
```

Comment:
nit: doesn't this mean we'll have to change the docs highlighting for this? same for the sync custom client, right?

Reply:
Nice catch. I fixed docs.